### ISSUE TITLE
Not firing at all

### ISSUE THREAD

**Issue by mkitchingh** [aortbals/southwest-checkin#74]
First, outstanding app. I love tinkering with stuff like this. I tried to get it running in a VMware environment, but I ran into some sort of issue. I followed all the instructions, but there was no web interface running when I finished. I'm not sure why, but there was nothing listening of 80 or 443. I looked at the issues in GitHub, and found this fork (clone) https://github.com/echo0101/southwest-checkin that deploys it to Heroku. Wow, that was an easy process. Everything looks great. I verified emails were working using the forgot your password feature. The only problem is it did nothing at check in time. The GUI shows the correct information, and it showed it was going to check in at the correct time. It didn't do anything though. Nothing showed up in the Heroku logs. The Heroku Dyno was definitely online. Any idea what else I could look at? I have 2 more flights in the next month.

**Comment by rusd11** [comment]
Try deploying to Heroku using flyize's fork. https://github.com/flyize/southwest-checkin

Make sure that Sidekiq is running. Your Procfile should have two processes, one should be the web app and the other should be the sidekiq process. Flyize's Procfile is configured correctly.

**Comment by rusd11** [comment]
Also make sure that you have Mailgun setup and working properly, that has been an issue in the past.

**Comment by mkitchingh** [comment]
Will do. Mailgun is definitely working. I did the forgot password feature to confirm.

**Comment by mkitchingh** [comment]
It looks like I got it all working. See screenshot. The problem was the worker process was toggled the left, so it looks like it was disabled. As soon as I toggled it on, I get all the confirmations for the reservations I had added. So cool. If I had known it did that on adding a reservation I would have known it wasn't working before. Mailgun was also having a integration issue with Heroku for all new setups, but they got that fixed. Thanks for the help. 
![image](https://cloud.githubusercontent.com/assets/25916510/23334418/ccc2290a-fb63-11e6-8744-dc056154af8d.png)

**Comment by mkitchingh** [comment]
Does anyone know if this version https://github.com/flyize/southwest-checkin is maintained? I don;t see any way to contact flyize. I've really enjoyed playing with Heroku. I have my instance monitored by Uptime Robot (free), so that keeps the Dyno online. I added the 'Logentries' add on for some more in depth logging/troubleshooting capabilities. I'm working on the 'Trevor' add on to allow simple access to the Postgres DB. I want to be able to see if anyone random signed up for the app. I dumped the DB and restored it to look at it, but I would like something a little quicker. I would be happy to contribute back to the overall project, but I"m not much of a coder.

**Comment by flyize** [comment]
Yep, I'm actively using it, so I sync it back with the changes here.
